<?xml version="1.0" ?>
<launch>    
<arg name="robot_name" value=""/>
<!-- <group ns="$(arg robot_name)"> -->
  <param name="gazebo"                   value="false"    type="bool"/>
  <param name="gazebo_robot_name"        value="robotis_op3"/>
    
  <param name="offset_file_path"         value="$(find op3_manager)/config/offset.yaml"/>
  <param name="robot_file_path"          value="$(find op3_manager)/config/OP3.robot"/>
  <param name="init_file_path"           value="$(find op3_manager)/config/dxl_init_OP3.yaml"/>
  <param name="device_name"              value="/dev/ttyUSB0"/>
	
  <param name="/robotis/direct_control/default_moving_time"     value="0.04"/>
  <param name="/robotis/direct_control/default_moving_angle"    value="90"/>
    
  <!-- OP3 Manager -->
  <node pkg="op3_manager" type="op3_manager" name="op3_manager" output="screen">
    <param name="angle_unit" value="30" />
  </node>

  <!-- OP3 Localization -->
  <node pkg="op3_localization" type="op3_localization" name="op3_localization" output="screen"/>

  <!-- Read Write -->
  <node pkg="soccer_pkg" type="read_write" name="op3_read_write" output="screen">
    <param name="gazebo_sim" value="false" type="bool"/>
    <param name="manager_name" value="/$(arg robot_name)/op3_manager"/>
  </node>

  <!-- Vision Node -->
   <!-- <node pkg="usb_cam" type="usb_cam_node" name="usb_cam" output="screen" />-->
  <!-- <node pkg="soccer_pkg" type="vision_node.py" name="op3_vision_node" output="screen" />-->

<!-- </group> -->
</launch>
